public/app/features/dashboard-scene/settings/annotations/AnnotationSettingsList.tsx

```diff
@@ -0,0 +1,135 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { AnnotationQuery } from '@grafana/data';
+import { getDataSourceSrv } from '@grafana/runtime';
+import { Button, DeleteButton, IconButton, useStyles2, VerticalGroup } from '@grafana/ui';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import { ListNewButton } from 'app/features/dashboard/components/DashboardSettings/ListNewButton';
+
+type Props = {
+  annotations: AnnotationQuery[];
+  onNew: () => void;
+  onEdit: (idx: number) => void;
+};
+
+export const AnnotationSettingsList = ({ annotations, onNew, onEdit }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  const onMove = (idx: number, direction: number) => {
+    console.log('todo: onMove');
+  };
+
+  const onDelete = (idx: number) => {
+    console.log('todo: onDelete');
+  };
+
+  const showEmptyListCTA = annotations.length === 0 || (annotations.length === 1 && annotations[0].builtIn);
+
+  const getAnnotationName = (anno: AnnotationQuery) => {
+    if (anno.enable === false) {
+      return (
+        <>
+          <em className="muted">(Disabled) &nbsp; {anno.name}</em>
+        </>
+      );
+    }
+
+    if (anno.builtIn) {
+      return (
+        <>
+          <em className="muted">{anno.name} &nbsp; (Built-in)</em>
+        </>
+      );
+    }
+
+    return <>{anno.name}</>;
+  };
+
+  const dataSourceSrv = getDataSourceSrv();
+  return (
+    <VerticalGroup>
```

Comment:
can't we reuse the previous component, making it agnostic to the received data? This component doesn't need to depend on any specific scene object, if I'm not wrong, we could create the AnnotationSettingList as an agnostic component with the callbacks to be used by `AnnotationEditView` and the component currently deployed.

Reply:
I would suggest we unify the list experiences between Variables (consider this a benchmark view), and Annotations and Links. Overall, Links and Annotations now use the "legacy" view, without drag and drop, while the Variables is probably the easiest to use currently. 

That unification would be a good move towards unified experience in the settings.

Reply:
I agree, but let leave this for later, once we have all the lists up and running. We could have something like

```jsx
<VariableList model> // Specific to scenes or redux based views
  <VariableList variables on... on... on...> // Reused between scenes and redux based views
    <SettingsTableList > // Reused between variables, annotations, and links
```

The settings list could also use the new Saga table, to remove boiler plating and custom table styles. But I'd suggest leaving this change once all the views are running. It will be easier to see patterns between them, also with all tests in place will be easier to refactor.

Reply:
Ok, captured this as a Dashboard Settings todo in https://github.com/grafana/grafana/issues/78389